docs/How-to-test-the-application.md

```diff
@@ -0,0 +1,33 @@
+Every time a new commit is merged into the develop branch an apk is released in the apk branch.To Navigate to apk Branch see here : 
+
+![Branch chnage](./images/branchOptionCrop.png)
+
+If you want to do manual testing on the application without setting up the development environment, you can download this app-debug.apk and install it on your Android device.
+
+![apk-beanch](./images/apkBranch.png)
+
+![apkInstall](./images/apkInstallMain.png)
+
+For manually testing in physical device tick the option here :
+![Screenshot of Unknown install option ](https://static.apkpure.com/www/static/imgs/unknown_sources.jpg)
+
+Now click the apk to install !
+
+For Testing on Emulator :
+
+- Now Open the project in Android Studio
+- Wait untill all background process are finished 
+- click the ADV button or click from already created devices in Android studio click 
+![advPress](./images/openADV.png)
+- For information on how to create an Emulator and Run it visit [Creating an ADV](https://developer.android.com/studio/run/managing-avds#createavd)
+- Now Run the Emulator you have created 
+![RunADV](./images/ADVButtonPress.png)
+- Navigate to the apk file downloaded 
+![apkAndEmulator](./images/apkAndEmulator.png)
+- Drag and Drop the Apk file into the Emulator.
+- Now just look for the App in App tray and click to open.
+![gif](./images/gifDemo.gif)
+- Now Test the app as required. :)
+
+If you want ideas of test cases to use when testing the mobile application business logic, you can check this set of Quality Assurance test casespage from systers/mentorship-backend repository.
```

Comment:
@TreavVasu please change systers/mentorship-backend to `anitab-org/mentorship-backend` and casespage to `cases page` and add links to both backend repo and QA test cases page